data/json/professions.json

```diff
@@ -2384,20 +2576,21 @@
         "items": [
           "bunker_pants",
           "bunker_coat",
+          "nomex_suit",
           "nomex_socks",
           "boots_bunker",
+          "nomex_gloves",
           "fire_gauntlets",
           "glasses_safety",
+          "nomex_hood",
           "firehelmet",
-          "pocketwatch"
```

Comment:
I got the impression the pocket watch was given to the firefighter because wearing a battery-powered watch into an area with high heat is a bad idea.

Reply:
Don't pocket watches use batteries too? Well, I reverted the change to this and the EMT firefighter.